backtester/eventhandlers/portfolio/size/size.go

```diff
@@ -67,14 +67,17 @@ func (s *Size) SizeOrder(o order.Event, amountAvailable float64, cs *exchange.Se
 // that is allowed to be spent/sold for an event.
 // As fee calculation occurs during the actual ordering process
 // this can only attempt to factor the potential fee to remain under the max rules
-func (s *Size) calculateBuySize(price, availableFunds, feeRate float64, minMaxSettings config.MinMax) (float64, error) {
+func (s *Size) calculateBuySize(price, availableFunds, feeRate, buyLimit float64, minMaxSettings config.MinMax) (float64, error) {
 	if availableFunds <= 0 {
 		return 0, errNoFunds
 	}
 	if price == 0 {
 		return 0, nil
 	}
 	amount := availableFunds * (1 - feeRate) / price
+	if buyLimit > minMaxSettings.MinimumSize && buyLimit <= minMaxSettings.MaximumSize && buyLimit <= amount {
```

Comment:
This looks like it should be
```
if buyLimit >= minMaxSettings.MinimumSize && buyLimit <= minMaxSettings.MaximumSize && buyLimit <= amount {
```
I see no reason why the buy limit can't be used if its the same as the minimum